solutions/projectstaffing/jgraph/core/src/main/scala/com/microsoft/graphdataconnect/skillsfinder/config/MdcFilter.scala

```diff
@@ -19,33 +19,42 @@ import org.springframework.context.annotation.Configuration
 @Configuration
 class MdcFilter(@Autowired override val userService: UserService) extends UserIdExtractionFilter(userService) {
   private val log: Logger = LoggerFactory.getLogger(classOf[MdcFilter])
+  private val arrayAnonymousUserInfo = Map("accesToken" -> "", "idToken" -> "", "refreshToken" -> "", "userId" -> "test@anonymous.com")
```

Comment:
Why did you define this entire map, since you only used one field from it? Why not just define an anonymousUserDefaultEmail instead?

Reply:
Also, please choose appropriate names for variables